### PR TITLE
Fill gaps in the RHEL8/RHEL9 STIG

### LINKED ISSUE #8643: Double check RHEL8 STIG missing rules from CaC/content to expand SRG coverage
The following RHEL8 STIG ids are reported as missing a rule in CaC/content, to make sure that the SRG mapping is complete we need to create at least the `rule.yml` file containing the basic information on how to fix and check.

Data extracted from this table: https://complianceascode.github.io/content-pages/tables/table-rhel8-stig.html

`
RHEL-08-010090 RHEL-08-010100 RHEL-08-010121 RHEL-08-010200 RHEL-08-010201 RHEL-08-010385 RHEL-08-010572 RHEL-08-020011 RHEL-08-020013 RHEL-08-020014 RHEL-08-020015 RHEL-08-020017 RHEL-08-020018 RHEL-08-020019 RHEL-08-020020 RHEL-08-020021 RHEL-08-020023 RHEL-08-020025 RHEL-08-020026 RHEL-08-020027 RHEL-08-020028 RHEL-08-020082 RHEL-08-020102 RHEL-08-020103 RHEL-08-020332 RHEL-08-030590 RHEL-08-040020 RHEL-08-040090 RHEL-08-040137 RHEL-08-040150 RHEL-08-040260
`

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

* Add new rules for gaps in the RHEL8/RHEL9 STIG

#### Rationale:

Fixes #8643

### CODE REVIEW

File: linux_os/guide/system/permissions/restrictions/kernel_module_uvcvideo_disabled/rule.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    If the camera is not disconnected, covered, or physically disabled, determine if it is being disabled via software with the following commands:
+
+    Verify the operating system disables the ability to load the uvcvideo kernel module.
+
+    $ sudo grep -r uvcvideo /etc/modprobe.d/* | grep "/bin/true"
+
+    install uvcvideo /bin/true
+
+fixtext: |-
+    Configure {{{ full_name }}}to disable the built-in or attached camera when not in use.
```
Comment (jan-cerny) [inline linux_os/guide/system/permissions/restrictions/kernel_module_uvcvideo_disabled/rule.yml:46]:
```suggestion
    Configure {{{ full_name }}} to disable the built-in or attached camera when not in use.
```

**Comment by ggbecker** [comment]
we need to respect some reasonable line length for these new rules... don't we have already something defined as maximum line length?

**Comment by jan-cerny** [comment]
@ggbecker What line length?

File: linux_os/guide/system/accounts/accounts-pam/disallow_bypass_password_sudo/rule.yml [hunk 2]
```diff
@@ -37,4 +38,14 @@ ocil: |-
     escalation. Check the configuration of the "/etc/pam.d/sudo" file with the following command:
     <pre>$ sudo grep pam_succeed_if /etc/pam.d/sudo</pre>
 
+fixtext: |-
+    Configure the operating system to require users to supply a password for privilege escalation.
+
+    Check the configuration of the "/etc/ pam.d/sudo" file with the following command:
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-pam/disallow_bypass_password_sudo/rule.yml:44]:
```suggestion
    Check the configuration of the "/etc/pam.d/sudo" file with the following command:
```

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**